chrome/updater/updater_module.cc

```diff
@@ -266,10 +266,17 @@ void UpdaterModule::Update() {
     return;
   }
 
+#if defined(COBALT_BUILD_TYPE_GOLD)
+  bool skipVerifyPublicKeyHash = false;
+#else  // defined(COBALT_BUILD_TYPE_GOLD)
```

Comment:
modify to: !defined(COBALT_BUILD_TYPE_GOLD)

Reply:
Do you mean for the comment after the #else? That make sense to negate it but in other places in our code we use the same condition as the #if, example: https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:cobalt/renderer/rasterizer/egl/textured_mesh_renderer.cc;l=809;drc=0edf22f6b8cf4203f0502749553449a2c4e0787c

Reply:
yup, just that one(after the else)

Reply:
There are a lot of [other examples](https://source.corp.google.com/search?q=%22%23else%20%20%2F%2F%20defined%22&sq=git:lbshell-internal%2Fcobalt_src@COBALT) of the same pattern in our code. I only looked at the first page of results but they all follow the pattern of repeating the statement in the #if.

I'll keep it this way to match the most common pattern in our code.